buildenv/jenkins/openjdk_tests

```diff
@@ -303,8 +303,11 @@ timestamps{
             }
 
             // JDK23+ isn't supported on machines prior to Mac 11 https://github.com/eclipse-openj9/openj9/issues/19694
-            if (params.JDK_IMPL == "openj9" && PLATFORM == "x86-64_mac" && params.JDK_VERSION && params.JDK_VERSION.toInteger() >= 23) {
-                LABEL += "&&!sw.os.mac.10"
+            // assume if JDK_VERSION is set to next, JDK_VERSION is 23+
+            if (params.JDK_IMPL == "openj9" && PLATFORM == "x86-64_mac" && params.JDK_VERSION) {
+                if (params.JDK_VERSION.equalsIgnoreCase("next") || params.JDK_VERSION.toInteger() >= 23) {
```

Comment:
I think this should be using `isInteger()` rather than make a special case for "next".
```
                if (!params.JDK_VERSION.isInteger() || params.JDK_VERSION.toInteger() >= 23) {
```

Reply:
updated